<launch> 
    <node pkg="assignment1" type="apriltag_locator" name="apriltag_locator"/>
    <node pkg="assignment1" type="node_B" name="node_B"/>
</launch>
